knife-azure.gemspec

```diff
@@ -21,10 +21,10 @@ Gem::Specification.new do |s|
 
   s.add_dependency "nokogiri", ">= 1.5.5"
   s.add_dependency "knife-windows", "~> 1.0"
-  s.add_dependency "azure_mgmt_resources", ">= 0.9.0"
-  s.add_dependency "azure_mgmt_compute", ">= 0.9.0"
-  s.add_dependency "azure_mgmt_storage", ">= 0.9.0"
-  s.add_dependency "azure_mgmt_network", ">= 0.9.0"
+  s.add_dependency "azure_mgmt_resources", "~> 0.17.0"
```

Comment:
@Vasu1105 Can we make these all >= requirements so we don't introduce constraints with other things in our stack that use these same libraries

Reply:
@tas50 did the changes.